roles/openshift_metering/tasks/main.yaml

```diff
@@ -1,6 +1,30 @@
 ---
-- include_tasks: install.yaml
-  when: openshift_metering_install | default(true) | bool
+- name: Openshift Metering RDS Database setup
+  import_tasks: rds-database-install.yaml
+  when: openshift_metering_hive_metastore_db_rds_create | bool
+  tags: [metering-hive-rds, metering-hive-rds-install]
 
-- include_tasks: uninstall.yaml
-  when: not openshift_metering_install | default(true) | bool
+- name: Openshift Metering S3 Bucket setup
+  import_tasks: s3-bucket-storage-install.yaml
+  when: openshift_metering_s3_bucket_create | bool
+  tags: [metering-s3-storage, metering-s3-storage-install]
+
+- name: Openshift Metering Operator installation
+  include_tasks: install.yaml
+  when: openshift_metering_install | bool
+  tags: metering-install
+
+- name: Openshift Metering Operator teardown
+  import_tasks: uninstall.yaml
```

Comment:
Move uninstall portions to their own play.

Reply:
Done